admin/nodes/views.py

```diff
@@ -467,3 +470,19 @@ def post(self, request, *args, **kwargs):
                                     ' if the problem persists get a developer to fix it.')
 
         return redirect(reverse_node(self.kwargs.get('guid')))
+
+
+class RemoveStuckRegistrationsView(StuckRegistrationsView):
+    template_name = 'nodes/remove_registrations_modal.html'
+
+    def post(self, request, *args, **kwargs):
+        from osf.management.commands.force_archive import verify
+        stuck_reg = self.get_object()
+        if verify(stuck_reg):
```

Comment:
This makes it so that stuck registrations with addons or added logs can't be removed.

I think the correct thing to do is 

```python
Registration.find_failed_registrations().filter(id=stuck_reg.id)).exists():
```

Reply:
I'll hotfix this.